lib/data/copy/stats.yml

```diff
@@ -1,4 +1,6 @@
-invalid: You're not a club leader! I currently only know how to give stats to club leaders.
+invalid:
+    leader: You don't seem to be a club leader! I can't get any stats for you :(
+    check_ins: You haven't had enough meetings for there to be any stats for you yet :(
```

Comment:
Let's change this to "Once you've had 3 meetings, I can show you a graph of your attendance".